content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml

```
@@ -34,8 +34,8 @@ Dimensions:
   Length: 80 mm
   Height: 90 mm
 IP Protection: IP20
-RTC: ~10days, NTP sync through ethernet
+RTC: ~10days, NTP sync through Ethernet or Wifi
```

Would it be more clear to state as follows:
```suggestion
RTC: ~10days, NTP sync through Wi-Fi® (AFX00002 only) or Ethernet
```

---

I like it, accepted, thanks!